src/exiv2.1

```diff
@@ -278,12 +278,47 @@ not rename the file (overrides \fB\-k\fP). This option is only used
 with the 'rename' action. Note: On Windows you may have to set the TZ
 environment variable for this option to work correctly.
 .TP
-.B \-f
-Do not prompt before overwriting existing files (force overwrite).
+.B \-f,\-F
```

Comment:
are used by commands 'rename' and 'extract' to determine file overwrite -> are used by the commands 'rename' and 'extract' to determine the file overwrite
('the' missing 2 times)